docs/creating/index.md

```diff
@@ -6,14 +6,24 @@ The model specifies one or more **nodes** along with the technologies allowed at
 **Demand technologies** remove carriers from the system through a **sink**, while **storage technologies** can store carriers at a specific node.
 Putting all of these possibilities together allows a modeller to specify as simple or as complex a model as necessary to answer a given research question.
 
+![Visual description of the Calliope terminology.](../img/description_of_system.svg)
+
 ??? info "An overview of the Calliope terminology"
-    The terminology defined here is used throughout the documentation and the model code and configuration files:
+    The terminology defined here is used throughout the documentation, and the model code and configuration files:
+
+    * **Carrier**: a commodity that is tracked inside the system boundary and can be produced or consumed by a _technology_.
+    > E.g., electricity, heat, hydrogen, water, CO<sub>2</sub>.
+    * **Technology**: an entity that produces, consumes, converts, stores or transports _carriers_.
+    > E.g., a _wind turbine_ that produces electricity, a _heat pump_ that converts electricity to heat, or a _battery_ that stores electricity.
+    * **Node**: a geographic point or area which can contain multiple _technologies_ and at which _carrier_ flows in and out must balance.
+    > E.g., a building in a district, an administrative region in a country, a country in a continent.
+    * **Source**: a commodity that can (or must) be used by a _technology_ to introduce carriers into the system.
+    Unlike _carriers_, _sources_ are not explicitly tracked as they are outside the system boundary.
+    > E.g., energy from the sun to power a solar panel, natural gas to power a gas turbine.
+    * **Sink**: a commodity that can (or must) be used by a _technology_ to remove carriers from the system.
+    Unlike _carriers_, _sinks_ are not explicitly tracked as they are outside system boundary.
+    > E.g., electricity consumed by household appliances, liquid fuel consumed by vehicles.
```

Comment:
This section is a better explanation than the text that starts this section (In short, a Calliope model works....).

How about trimming that text, and making it a  friendlier introduction?

----

A Calliope model is a collection of interconnected technologies, nodes and carriers describing a real world energy system. **Technologies** can be anything in between transmission lines, storage facilities, power supplies or energy demanding appliances. **Nodes** are groupings of technologies describing a region of the world, which could be a country, municipality or a single house. Finally, **carriers** describe the different types of energy being produced, consumer or transmitted. This could be electricity, heat or fuels needed for all kinds of different activities.

Putting all of these possibilities together allows a modeller to create a model that is as simple or as complex as necessary to answer a given research question, and to do so in a way that is easy to understand to other researchers thanks to Calliope's straightforward syntax.

Reply:
I like the more straightforward version you propose, @irm-codebase, but I guess that the original one is more generalisable as in there is no mention of specific energy-related terms such as 'lines', 'power supplies', 'energy', etc. This is possibly meant to keep open the option to model _whatever_ system in Calliope, which makes sense since it has already happened several times.

I would try to strike a balance between too-generic (and therefore, not straightforward) descriptions and too-energy-focussed ones. Perhaps one solution would be to make it explicit that the framework is generalised to to other systems and flows than energy ones, but we make an example description for an energy system as the most typical application.

Reply:
Danger is that there's repetition, right?

Reply:
Hey, repetitive or not, I love these changes!

Reply:
@brynpickering yep, repetition would be the main issue. I think we should aim to be as concise as possible with the docs, to avoid leaving old stuff as the software changes.

Reply:
OK, I've gone with what is hopefully both more concise and clearer.